tabix.1

```diff
@@ -78,6 +78,8 @@ and BAI index formats can handle individual chromosomes up to 512 Mbp
 If your input file might contain data lines with begin or end positions
 greater than that, you will need to use a CSI index.
 
+A thread pool can be used for operations except listing sequence names.
```

Comment:
Suggestion, change "thread pool" to "threads".  "thread pool" is more of an internal name.

Reply:
Also, do the same in the other places in the man file where "thread pool" is used.

Reply:
Updated and rephrased.